config.js

```diff
@@ -92,6 +92,20 @@ var config = {
     // Sets the preferred resolution (height) for local video. Defaults to 720.
     // resolution: 720,
 
+    // New style constraints to use for video capture. Currently supported fro
+    // chrome only.
+    constraints: {
```

Comment:
Please don't put these uncommented. Leave them commented and make sure we have defaults inplace if set to undefined. Also, I think we should only accept video contraints, and not accept anything audio.